scripts/tmp.R

```diff
@@ -0,0 +1,120 @@
+library(PLIER)
```

Comment:
Looks like this file is not used and should probably be removed.

Reply:
yes, I was testing some analysis only!